.bra.toml

```diff
@@ -13,7 +13,7 @@ watch_dirs = [
   "$WORKDIR/conf",
 ]
 watch_exts = [".go", ".ini", ".toml", ".template.html"]
-ignore_files = ["wire_gen.go"]
```

Comment:
Local env was broken and added this changes until the other PR is merged. Will need to remove/rebase before merging